apps/storybook/src/tab.stories.tsx

```diff
@@ -6,23 +6,30 @@ const meta: Meta<typeof Tab> = {
   args: {
     size: "md",
     orientation: "horizontal",
+    variant: "line",
   },
   argTypes: {
     size: { control: "select", options: ["sm", "md", "lg"] },
+    variant: { control: "select", options: ["enclosed", "line"] },
   },
 };
 
 export default meta;
 type Story = StoryObj<typeof Tab>;
 
 export const Variants: Story = {
-  render: ({ size, orientation }) => (
+  render: ({ size, orientation, variant }) => (
     <>
       <div
         data-cy="tab-1"
         className="w-[640px] rounded-md border py-2 dark:border-zinc-700"
       >
-        <Tab defaultValue="tab1" size={size} orientation={orientation}>
+        <Tab
```

Comment:
- Remove 2nd tab and pass all args props to 1st tab
- Also remove line that says variant=enclosed